lib/order.rb

```diff
@@ -0,0 +1,99 @@
+require "csv"
+
+class Order
+  attr_reader :id
+  attr_accessor :products, :customer, :fulfillment_status
+
+  def initialize(id, products, customer, fulfillment_status = :pending)
+    @id = id
+    @products = products
+    @customer = customer
+
+    fulfillment_options = [:pending, :paid, :processing, :shipped, :complete]
+
+    unless fulfillment_options.any? fulfillment_status
```

Comment:
Nice implementation here! maybe `fulfillment_options` can even be a constant variable.